### ISSUE TITLE
Learning process is extremely slow

### ISSUE THREAD

**Issue by brandtbucher** [brandtbucher/SpeakEasy#2]
When cleared, the program is taking much longer to recognize and use phrases than it should.

**Comment by brandtbucher** [comment]
It is caused by lines 145...

```python
                if phrase_b not in phrases[phrase_a]: phrases[phrase_a][phrase_b] = 0
```

...and 157...

```python
                    if phrase_b not in context[phrase_a]: context[phrase_a][phrase_b] = 0
```

...which require phrases to be used *twice* before they begin appearing in output text, rather than once.

**Comment by brandtbucher** [comment]
Fixed in `1.0.1`.

`145` has been replaced with...

```python
                if phrase_b not in phrases[phrase_a]: phrases[phrase_a][phrase_b] = rating ###
```

...and '157' has been replaced with:

```python
                    if phrase_b not in context[phrase_a]: context[phrase_a][phrase_b] = rating ###
```
